README.md

```
@@ -211,6 +211,38 @@ dotnet ef database update --project PatchPanda.Web --startup-project PatchPanda.
 
 - Notifications are posted to the configured Discord webhook and Apprise notification URLs. PatchPanda will chunk long messages (Discord limits message length) and mark versions as notified once it posted them.
 
+## Post-hooks
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)